CHANGELOG.md

```diff
@@ -59,7 +59,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
-- Update `upstream chart` to [kube-prometheus-stack-32.4.0](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-32.4.0). See [breaking changes](./changelog/23.x_32.x.md)
+- Update `upstream chart` to [kube-prometheus-stack-32.4.0](https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-32.4.0). See [breaking changes](https://github.com/giantswarm/prometheus-operator-app/blob/master/README.md#from-1.x-to-2.x)
```

Comment:
Correct link: https://github.com/giantswarm/prometheus-operator-app/blob/main/README.md#from-13x-to-2x